api/src/AppBundle/v2/Controller/CourtOrderController.php

```diff
@@ -66,15 +69,25 @@ public function __construct(
      * @throws ORMException
      * @throws OptimisticLockException
      */
-    public function createAction(Request $request): JsonResponse
+    public function createAction(
+        Request $request,
+        LayToCourtOrderDeputyDtoAssembler $deputyAssembler,
+        CourtOrderDeputyFactory $deputyFactory
+    ): JsonResponse
     {
         $data = json_decode($request->getContent(), true);
         $client = $this->clientRepository->find($data['id']);
+        $user = $this->getUser();
         $registrationData = $this->casRecRepository->findOneBy(['caseNumber' => $client->getCaseNumber()]);
 
         $courtOrderDto = $this->courtOrderAssembler->assemble($registrationData);
         $courtOrder = $this->courtOrderFactory->create($courtOrderDto, $client, $client->getCurrentReport());
 
+        $deputyDto = $deputyAssembler->assemble($registrationData, $user);
+        $deputy = $deputyFactory->create($deputyDto, $courtOrder);
+
+        $deputy->setUser($user);
+
```

Comment:
Should we call `$this-em->persist($deputy)`?

Reply:
We don't need to, since the CourtOrder has `cascade={"persist", "remove"}` on the relationship. I was a bit iffy on this though since that's not clear in situ (as you've identified!). Happy to fire another persist in if you think it'd be clearer.

Reply:
I'm happy with the cascade operation 👍